libs/CMakeLists.txt

```diff
@@ -84,29 +79,44 @@ function(add_external_project name)
   endif()
 
   ExternalProject_Add(${name}-proj DEPENDS ${KW_DEPENDS}
-    PREFIX ${source_dir} SOURCE_DIR ${source_dir}
-    BUILD_IN_SOURCE ${build_in_source}
-    CONFIGURE_COMMAND ${KW_CONFIGURE_COMMAND}
-    CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release
-    -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
-    -DCMAKE_INSTALL_PREFIX=${source_dir}
-    ${KW_CMAKE_ARGS}
-    INSTALL_COMMAND ${KW_INSTALL_COMMAND}
-    BUILD_COMMAND ${KW_BUILD_COMMAND})
+          PREFIX ${source_dir} SOURCE_DIR ${source_dir}
+          BUILD_IN_SOURCE ${build_in_source}
+          LIST_SEPARATOR " "
+          CONFIGURE_COMMAND ${KW_CONFIGURE_COMMAND}
+          CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release
+          -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+          -DCMAKE_INSTALL_PREFIX=${source_dir}
+          ${KW_CMAKE_ARGS}
+          INSTALL_COMMAND ${KW_INSTALL_COMMAND}
+          BUILD_COMMAND ${KW_BUILD_COMMAND})
```

Comment:
Note that by doing this, you suppress the default command generation for CMake projects. If everything is CMake, I'd recommend just not mentioning these at all.